### PR TITLE
PTNFLY-1570 List View full page implementation to be consumed by the …

### PR DESCRIPTION
**Pull request by cardosogabriel**
…patternfly.org.

### CODE REVIEW

**Comment by cardosogabriel** [comment]
@jeff-phillips-18 , @dlabrecq , @dtaylor113 , @andresgalante , can you review this one?

File: tests/pages/list-view.html [hunk 1]
```diff
@@ -1,16 +1,11 @@
...
-categories: [Widgets]
-layout: page
-title: List View
+categories: [Layouts]
+css-extra: false
+layout: layout
 resource: true
+full-page: true
+title: List View
 url-js-extra: ['//cdnjs.cloudflare.com/ajax/libs/c3/0.4.10/c3.min.js', '//cdnjs.cloudflare.com/ajax/libs/d3/3.5.0/d3.min.js']
```
Comment (jeff-phillips-18) [inline tests/pages/list-view.html]: Since the rows are not using charts, I don't believe the url-js-extra is necessary

Comment (cardosogabriel) [inline tests/pages/list-view.html]: Nice catch, I removed the line. Thank you!

**Comment by cardosogabriel** [comment]
This PR is to be consumed by the PR in patternfly-org: https://github.com/patternfly/patternfly-org/pull/227

**Comment by jeff-phillips-18** [comment]
👍

**Comment by bleathem** [comment]
I built this branch, and compared the test page: http://localhost:9000/list-view-rows.html to the design page on http://www.patternfly.org/pattern-library/content-views/list-view.  Visually everything checks out.

The code committed seems reasonable.  Looks ready to merge to me.

:+1:

**Comment by jeff-phillips-18** [comment]
@cardosogabriel Please rebase so this can be merged

**Comment by cardosogabriel** [comment]
@bleathem , thanks for the review. 
@jeff-phillips-18 , it is rebased now.

**Pull request merged**